ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
+        """
+        
+        Parameters
+        -------
+        columns: list of eligible column names
+        logic: str, optional, "any" by default. If logic is set to "any" then all rows which contain a NaN value in any 
+        (id est at least one) of the cited columns is deleted, this is a union condition; 
+        if logic is set to "all", then only rows containing null values in all the fields are deleted, this is an intersection condition.         
+        inplace: bool, optional, False by default. If True, then the underlying table is physically modified,
+        if False, then the original objects remain unaffected, a copy of the IdaDataFrame is made and modified. 
+        
+        Returns
+        -------
+        If inplace is True, the original table is modified and the IdaDataFrame will be modified accordingly. No return.
+        If inplace is False, a new IdaDataFrame is returned, the original table is not affected. This new IdaDataFrame points to a 
+        new table which has been created according to the user defined criterions on columns.
+        
+        Examples
+        -------
+        idadf.delete_na(["COL_1", "COL_2"], inplace = True)
+        # each row of the original table is physically deleted if there is a NULL value in one of the listed columns
+        new_idadf = idadf.delete_na(["COL_1", "COL_2"], union = False)
```

Comment:
This function doesn't have a parameter called `union`.

Reply:
this example was outdated, here is the update:
`new_idadf = idadf.delete_na(["COL_1", "COL_2"], logic = 'all')`